guides/_posts/2014-01-5-vfl.html

```diff
@@ -121,36 +133,37 @@
 
 <h2>Containment within an Element</h2>
 
-For alignment contained within an element, surround a VFL alignment with `|`'s and use `in( Selector )`:
+For alignment contained within an element, surround a VFL alignment with `|`'s and use `in( Selector )`. <a href="http://codepen.io/cbchouinard/pen/emEwRr" target="_blank">Live example.</a>
 
 <img class="vfl-small-img" src="../../assets/images/vfl/connectionToSuperview.png" width="152" height="32" alt="ConnectionToSuperview">
 
 {% highlight css %}
 /* VFL */
-@h |-50-[#purple-box]-50-| in(#container);
+@h |-50-(#purple-box)-50-| in(#container);
 
 /* Equivalent CCSS */
 #container[left] + 50 == #purple-box[left];
 #container[right] - 50 == #purple-box[right];
 {% endhighlight %}
 
-The default containing element selector is `::this`.  The above example can be written:
+The default containing element selector is `&`.
+
+The above example can be written:
 
 {% highlight css %}
 /* VFL */
 #container {
-  @h |-50-[#box]-50-|;
+  @h |-50-(#box)-50-|;
 }
 
 /* Equivalent CCSS */
 #container {
-  ::[left] + 50 == #purple-box[left];
-  ::[right] - 50 == #purple-box[right];
+  &[left] + 50 == #purple-box[left];
+  &[right] - 50 == #purple-box[right];
 }
```

Comment:
Might be worth mentioning that it's not necessary in this case, since we can just do this:

```
#container {
  left: == #purple-box[left] - 50;
  right: == #purple-box[right] + 50;
 }
```

Not sure if we can omit the colon here.